packages/react/src/drafts/SelectPanel2/SelectPanel.examples.stories.tsx

```diff
@@ -861,20 +859,22 @@ export const WithinForm = () => {
     <>
       <h1>Within Form</h1>
 
-      <FormControl>
-        <FormControl.Label>SelectPanel within FormControl</FormControl.Label>
-        <SelectPanel title="Choose a tag" selectionVariant="instant" onSubmit={onSubmit}>
-          <SelectPanel.Button leadingVisual={TagIcon}>{selectedTag || 'Choose a tag'}</SelectPanel.Button>
+      <form>
```

Comment:
It doesn't seem like this is triggering the error

> `<form>` cannot appear as a descendant of `<form>`.

Do we know why this is only occurring in Dotcom and not Storybook and if there's a way to achieve parity between the two to avoid future problems?

Reply:
Shouldn't this error have been showing without needing to wrap `<form>` around `<FormControl>`? Do we know why it was occurring in Dotcom and not Storybook and if there's a way to achieve parity between the two to avoid future problems?